style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
As a way to set some defaults for your whole HTML document, I recommend that every `body` ruleset include the following:
- `background` or `background-color`
- `color`
- `font-family`

You can always change these defaults on individual elements or specific classes elsewhere in your stylesheet, but adding them here creates a solid foundation and will make sure your document displays as expected across browsers.